# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../asterisk-addons/asterisk-addons-1.4.13-mysql-5.5-header.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN asterisk-addons-1.4.13-orig/apps/app_addon_sql_mysql.c asterisk-addons-1.4.13/apps/app_addon_sql_mysql.c
--- asterisk-addons-1.4.13-orig/apps/app_addon_sql_mysql.c	2010-10-08 16:42:56.000000000 +0200
+++ asterisk-addons-1.4.13/apps/app_addon_sql_mysql.c	2011-03-19 00:09:11.964072237 +0100
@@ -26,7 +26,7 @@
 #include <stdio.h>
 #include <unistd.h>
 
-#include <mysql/mysql.h>
+#include <mysql.h>
 
 #include <asterisk/file.h>
 #include <asterisk/logger.h>
diff -ruN asterisk-addons-1.4.13-orig/cdr/cdr_addon_mysql.c asterisk-addons-1.4.13/cdr/cdr_addon_mysql.c
--- asterisk-addons-1.4.13-orig/cdr/cdr_addon_mysql.c	2010-10-08 16:42:56.000000000 +0200
+++ asterisk-addons-1.4.13/cdr/cdr_addon_mysql.c	2011-03-19 00:10:45.840085556 +0100
@@ -41,8 +41,8 @@
 #include <unistd.h>
 #include <time.h>
 
-#include <mysql/mysql.h>
-#include <mysql/errmsg.h>
+#include <mysql.h>
+#include <errmsg.h>
 
 #include <sys/stat.h>
 #include <sys/types.h>
diff -ruN asterisk-addons-1.4.13-orig/res/res_config_mysql.c asterisk-addons-1.4.13/res/res_config_mysql.c
--- asterisk-addons-1.4.13-orig/res/res_config_mysql.c	2010-10-08 16:42:56.000000000 +0200
+++ asterisk-addons-1.4.13/res/res_config_mysql.c	2011-03-19 00:09:11.964072237 +0100
@@ -56,9 +56,9 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdio.h>
-#include <mysql/mysql.h>
-#include <mysql/mysql_version.h>
-#include <mysql/errmsg.h>
+#include <mysql.h>
+#include <mysql_version.h>
+#include <errmsg.h>
 
 AST_MUTEX_DEFINE_STATIC(mysql_lock);
 #define RES_CONFIG_MYSQL_CONF "res_mysql.conf"
